packages/fiori/test/pages/ViewSettingsDialog.html

```diff
@@ -88,11 +88,20 @@ <h3> ViewSettingsDialog</h3>
 		</ui5-filter-item>
 	</ui5-view-settings-dialog>
 
+	<br>
+	<br>
+	<ui5-input id="sortByItem" style="display: none"></ui5-input>
+	<ui5-input id="sortOrder" style="display: none"></ui5-input>
+
 	<script>
 		btnOpenDialog.addEventListener("click", function () {
 			vsd.show();
 		});
-		vsd.addEventListener("confirm", function(evt) {
+		vsd.addEventListener("ui5-confirm", function(evt) {
+			const sortByItemTagName = evt.detail.sortByItem && evt.detail.sortByItem.tagName;
```

Comment:
Let's stick to es5 for the moment as we do everywhere in the test pages
`const -> var`

I think `const isSortDescending` is not needed, you can just 
`sortOrder.value = evt.detail.sortDescending;`